cmd/neofs-node/config.go

```diff
@@ -308,9 +314,15 @@ type cfgControlService struct {
 }
 
 type cfgReputation struct {
+	workerPool util2.WorkerPool // pool for EigenTrust algorithm's iterations
+
 	localTrustStorage *truststorage.Storage
 
 	localTrustCtrl *trustcontroller.Controller
+
+	scriptHash util.Uint160
+
+	rptWrapper *rptwrapper.ClientWrapper
```

Comment:
hmm, i did it like it was done in containers, but forget to add init in `reputation` file. i think it's because i dont know should it be stored in cfg struct or not

i may add initialization in `reputation` file or just delete it. which one is better?

Reply:
I don't see usage of reputation contract wrapper outside of reputation components, so I think the better way is to remove wrapper from config.

Reply:
ok, deleted